packages/fx-core/src/component/driver/env/appsettingsGenerate.ts

```diff
@@ -17,7 +17,7 @@ import { InvalidParameterUserError } from "./error/invalidParameterUserError";
 import { UnhandledSystemError } from "./error/unhandledError";
 import { GenerateAppsettingsArgs } from "./interface/generateAppsettingsArgs";
 
-const actionName = "appsettings/generate";
+const actionName = "file/updateAppSettings";
```

Comment:
May conflict with #7117? Moving this file to `file/` folder

Reply:
Solve this by merge dev branch.